genai-function-calling/vercel-ai/mcp.js

```diff
@@ -1,16 +1,19 @@
-const {McpServer} = require('@modelcontextprotocol/sdk/server/mcp.js');
-const {StdioServerTransport} = require('@modelcontextprotocol/sdk/server/stdio.js');
-const {StdioClientTransport} = require('@modelcontextprotocol/sdk/client/stdio.js');
-const {experimental_createMCPClient} = require('ai');
+import { McpServer } from '@modelcontextprotocol/sdk/server/mcp.js';
+import { StdioServerTransport } from '@modelcontextprotocol/sdk/server/stdio.js';
+import { StdioClientTransport } from '@modelcontextprotocol/sdk/client/stdio.js';
+import { experimental_createMCPClient } from 'ai';
 
-const fs = require('fs');
-const path = require('path');
+import fs from 'fs';
+import path from 'path';
+import { fileURLToPath } from 'url';
 
 const SERVER_ARG = '--mcp-server';
 
 // Get MCP server parameters from package.json
 let name, version;
 try {
+    const __filename = fileURLToPath(import.meta.url);
```

Comment:
ack I might be digging a hole on this point and might just switch it to hard-coded since the values don't matter in an example